src/test/test_chunk_processors.py

```diff
@@ -12,49 +14,49 @@ def test_remove_short_table_cells_drop_numeric():
     chunks = [
         Chunk(
             text="short",
-            chunk_type=ChunkType.TABLE_CELL,
+            chunk_type=BlockType.TABLE_CELL,
```

Comment:
If a chunk can span multiple text blocks that can have different block types what do we do here?

Reply:
A chunk will only have one type I think – but multiple pages and sets of coordinates can be captured. I wanted to introduce a new `ChunkType` but I think that causes more problems than it solves atm